cdap-adapters/src/main/java/co/cask/cdap/conversion/app/StreamConversionMapReduce.java

```diff
@@ -93,6 +93,7 @@ public void beforeSubmit(MapReduceContext context) throws Exception {
     if (adapterArguments.getHeadersStr() != null) {
       job.getConfiguration().set(HEADERS_KEY, adapterArguments.getHeadersStr());
     }
+    job.setJobName("StreamConversion_" + adapterArguments.getSourceName() + "_" + sinkName + "-" + partitionTime);
```

Comment:
@albertshau should this be `stream-adapter.<source-name>.to.<sink>.<part>`. Just a suggestion.